apps/frontend/stores/current-repository.ts

```diff
@@ -82,6 +82,12 @@ export const useCurrentRepository = defineStore('currentRepository', () => {
     return outlineActivities.value.find((it) => it.id === id);
   });
 
+  const hasGuidelines = computed(() => {
```

Comment:
Since this is within current-repository store, naming it just hasGuideliness might imply this is repo level config.